util/importers.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import copy
+import importlib
+import os
+import re
+import sys
+
+from util.helpers import secs_to_hours
+
+def get_counter():
+    return copy.deepcopy({'all': 0, 'failed': 0, 'invalid_label': 0, 'too_short': 0, 'too_long': 0, 'total_time': 0})
+
+def get_imported_samples(counter):
+    return counter['all'] - counter['failed'] - counter['too_short'] - counter['too_long'] - counter['invalid_label']
+
+def print_import_report(counter, sample_rate, max_secs):
+    print('Imported %d samples.' % (get_imported_samples(counter)))
+    if counter['failed'] > 0:
+        print('Skipped %d samples that failed upon conversion.' % counter['failed'])
+    if counter['invalid_label'] > 0:
+        print('Skipped %d samples that failed on transcript validation.' % counter['invalid_label'])
+    if counter['too_short'] > 0:
+        print('Skipped %d samples that were too short to match the transcript.' % counter['too_short'])
+    if counter['too_long'] > 0:
+        print('Skipped %d samples that were longer than %d seconds.' % (counter['too_long'], max_secs))
+    print('Final amount of imported audio: %s.' % secs_to_hours(counter['total_time'] / sample_rate))
+
+def get_importers_parser(description):
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('--validate_label_locale', help='Path to a python module defining a |validate_label| function for your locale')
+    return parser
+
+def get_validate_label(args):
+    if 'validate_label_locale' not in args.__dict__ or (args.validate_label_locale is None):
+        print('WARNING: No --validate_label_locale specified, your might end with inconsistent dataset.')
+        return validate_label_eng
+    if not os.path.exists(os.path.abspath(args.validate_label_locale)):
+        print('ERROR: Inexistent --validate_label_locale specified. Please check.')
+        return None
+    module_dir = os.path.abspath(os.path.dirname(args.validate_label_locale))
+    sys.path.insert(1, module_dir)
+    fname = os.path.basename(args.validate_label_locale).replace('.py', '')
+    locale_module = importlib.import_module(fname, package=None)
+    return locale_module.validate_label
```

Comment:
Woah, this is some wild behavior, and completely undocumented. The importers are already doing `sys.path` modification which is broken enough, we should avoid adding to it. If the locale specific module/function is in a Python package, it should be installed, and then it can just be imported with `importlib` instead of modifying `sys.path` here.

Reply:
I'm unsure about forcing people to write and install a module, it's going to make it more complicated than just writing a simple `.py` file with `def validate_label(label)` inside.

Reply:
If you feel strongly about keeping this, then it should at least be documented.

Reply:
Okay, I'll try to figure proper doc.

Reply:
@reuben I've made doc-related improvement, does it looks nice to you ?

Reply:
Looks better, thanks.